src/main/resources/default/assets/wondergem/autocomplete/multiselect.js

```diff
@@ -215,6 +217,34 @@ var multiSelect = function (args) {
                 tokenfield.getTokenfieldInputField()[0].placeholder = args.searchKey || args.placeholder;
             });
 
+            $('#' + args.id).on('keyup', '.tokenfield', function (e) {
+                if (e.target !== e.currentTarget) {
+                    // only responde to key-events off the container
+                    return;
+                }
+                if (e.keyCode !== sirius.keys.KEY_ENTER
+                    && e.keyCode !== sirius.keys.KEY_ARROW_DOWN
+                    && e.keyCode !== sirius.keys.KEY_SPACE) {
+                    return;
+                }
+
+                // when a singleselect is selected via tab, then the keys enter, space and arrow down should open the
+                // autocomplete and make the field editable, just like clicking on the field
+                $('#' + args.id).find('.tokenfield').click();
+            });
+
+            $('#' + args.id).on('keydown', '.tokenfield', function (e) {
+                if (e.target !== e.currentTarget) {
+                    // only responde to key-events off the container
+                    return;
```

Comment:
return true?

Reply:
nvm in jquery ist es das gleiche > return true is the same as returning nothing (no action is taken by jQuery).